src/error.rs

```diff
@@ -16,7 +14,7 @@ impl Error {
 }
 
 impl fmt::Display for Error {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
What prompted you to do this change? I'm not against it, but the simpler version works and doesn't seem to have any downsides.

Reply:
It's rust_2018_idioms. Hidden anonymous lifetime are deprecated and will maybe be removed in the 2021 version.